pkg/dlx/handler.go

```diff
@@ -109,8 +108,11 @@ func (h *Handler) handleRequest(res http.ResponseWriter, req *http.Request) {
 			res.WriteHeader(http.StatusBadRequest)
 			return
 		}
+		// add the path addition to the path if the header exists
+		// this is used when the ingress controller needs to add a suffix to the path request
+		pathAddition := req.Header.Get(h.targetPathHeader)
 		for _, resourceName := range resourceNames {
-			targetURL, status := h.parseTargetURL(resourceName, path)
+			targetURL, status := h.parseTargetURL(resourceName, pathAddition)
```

Comment:
I see in https://github.com/v3io/scaler/pull/82/files#diff-1ccd3a45f2df3be1905f15b05090b6ae7f8d6630df4bb252946c7e212abc71e3R214 that the `path` is always added. Add a check that it isn't empty.